CreateAnalyticsServiceServiceDbTables.sql

```diff
@@ -1,3 +1,6 @@
+USE [AnalyticsServiceServiceDb]
```

Comment:
Scripts intentionally do not have USE statement, because it is not allowed in Azure SQL Database, where connection must be made directly to target database.
However it might be good solution to have script for preparing local deployment environment. It would execute all scripts in correct order against correct databases.